### PR TITLE
Port existing Mahalanobis Distance example to meta

### PR DESCRIPTION
**Pull request by abhinavrai44**

### CODE REVIEW

**Comment by abhinavrai44** [comment]
@karlnapf 
Hi,
Travis gives the error - > Fetched in submodule path 'data', but it did not contain b89c9939aed689378bbbdc43edaa6cdb535a03f6. Direct fetching of that commit failed.

Can you guide me as to what I am doing wrong. 

Thanks

**Comment by karlnapf** [comment]
This is because the data PR has not been merged yet. Doing that now

File: doc/cookbook/source/examples/distance/mahalanobis.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+======================
+Mahalanobis Distance
+======================
+
+The Mahalanobis distance for real valued features computes the distance between a feature vector and a distribution of features characterized by its mean and covariance.
+
+.. math::
+
+    D =  \sqrt{ ( x_{i} -  \mu  )^{T}   S^{-1} ( x_{i} -  \mu  )}
```
Comment (karlnapf) [inline doc/cookbook/source/examples/distance/mahalanobis.rst]: Please use `\top` for transpose

Comment (karlnapf) [inline doc/cookbook/source/examples/distance/mahalanobis.rst]: Please dont define D, just give the formula

**Comment by karlnapf** [comment]
In fact, travis is fine. If you can update the data, then I can merge this

**Comment by karlnapf** [comment]
The data version is still not updated to your merged data PR

**Comment by karlnapf** [comment]
Nice!

**Pull request merged**